lib/ansible/module_utils/aws/elb_utils.py

```diff
@@ -13,6 +15,22 @@ def get_elb(connection, module, elb_name):
     """
     Get an ELB based on name. If not found, return None.
 
+    :param connection: AWS boto3 elbv2 connection
+    :param module: Ansible module
+    :param elb_name: Name of load balancer to get
+    :return: boto3 ELB dict or None if not found
+    """
+    try:
+        return _get_elb(connection, module, elb_name)
+    except (BotoCoreError, ClientError) as e:
+        module.fail_json_aws(e)
+
+
+@AWSRetry.jittered_backoff()
```

Comment:
@s-hertel i'm not sure this works here.  If the ELB doesn't exist then the retry logic means it'll keep retrying causing a large delay even though it's quite likely we just want to move on and create the ELB.

Reply:
If I remove the `return None` and raise the exception again instead it does retry continuously (before failing since the `return None` is necessary): `Playbook run took 0 days, 0 hours, 4 minutes, 33 seconds`

Reply:
Hmm I had different behaviour over the weekend. Let me retry and get back to you

Reply:
Hmm I had different behaviour over the weekend. Let me retry and get back to you

Reply:
Can not replicate behaviour so let's sweep this under the carpet ;)